app/services/users/form_526_user_identification_verifier.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
```

Comment:
Just a suggestion, but it might make sense to take an approach where the list of identifiers can be dynamic. Something like this is what I had in mind:
```ruby
module Users
  class Form526UserIdentificationVerifier
    def initialize(user)
      @user = user
    end

    REQUIRED_IDENTIFIERS = %w[birls_id birth_date edipi participant_id ssn]

    def missing_identifiers
      REQUIRED_IDENTIFIERS.select { |id| @user[id].blank? }
    end
  end
end
```

Reply:
Looking at it again, you could also consider making `missing_identifiers` a static method and not need to instantiate this class if you wanted

Reply:
Yeah I considered both of these suggestions, I generally prefer the dynamic identifiers approach and did that originally but got sidetracked by the delegation thing, I think what you have above works so I might just return to that approach.

Thanks for taking a look!

Reply:
Fixed in https://github.com/department-of-veterans-affairs/vets-api/pull/13369/commits/60acea6d7a9aa68d071537a9bf42902c20dc536c, I had something like this originally but the syntax I had wasn't working as expected, I didn't realize `@user[id]` would work in this case, thanks for the tip:
`REQUIRED_IDENTIFIERS.select { |id| @user[id].blank? }`